src/ess/dream/io/geant4.py

```diff
@@ -101,10 +106,13 @@ def group(key: str, da: sc.DataArray) -> sc.DataArray:
         if key in ["high_resolution", "sans"]:
             # Only the HR and SANS detectors have sectors.
             res = da.group("sector", *elements)
+        elif key in ["endcap_backward", "endcap_forward"]:
+            res = da.group("sumo", *elements)
```

Comment:
Why only for endcaps?

Reply:
For the other banks (mantle, HR and SANS), sumo = 1